ci-operator/config/openshift-kni/eco-ci-cd/openshift-kni-eco-ci-cd-main__far-edge-vran-dast-4.16.yaml

```diff
@@ -0,0 +1,59 @@
+base_images:
+  base:
+    name: "4.16"
+    namespace: ocp
+    tag: base
+build_root:
+  image_stream_tag:
+    name: ci
+    namespace: openshift
+    tag: golang-1.20
+releases:
+  latest:
+    candidate:
+      product: ocp
+      stream: nightly
+      version: "4.16"
+resources:
+  '*':
+    limits:
+      memory: 4Gi
+    requests:
+      cpu: 100m
+      memory: 200Mi
+tests:
+- as: telco-dast-ci
+  cluster: build05
+  cron: 00 13 * * 1
+  steps:
+    cluster_profile: equinix-ocp-metal-qe
+    env:
+      ADDITIONAL_WORKER_ARCHITECTURE: x86_64
+      ADDITIONAL_WORKERS: "0"
+      ADDITIONAL_WORKERS_DAY2: "false"
+      AGENT_PLATFORM_TYPE: none
+      AUX_HOST: openshift-qe-metal-ci.arm.eng.rdu2.redhat.com
+      DISCONNECTED: "false"
+      OPERATORS: |
+        [
+          {"name": "local-storage-operator", "source": "redhat-operators", "channel": "stable", "install_namespace": "openshift-local-storage", "target_namespaces": "openshift-local-storage", "operator_group": "local-operator-group"},
+          {"name": "advanced-cluster-management", "source": "redhat-operators", "channel": "release-2.11", "install_namespace": "open-cluster-management", "target_namespaces": "open-cluster-management", "operator_group": "acm-operator-group"},
+          {"name": "topology-aware-lifecycle-manager", "source": "redhat-operators", "channel": "stable", "install_namespace": "openshift-talm", "target_namespaces": "", "operator_group": "topology-aware-lifecycle-manager"},
+          {"name": "openshift-gitops-operator", "source": "redhat-operators", "channel": "!default", "install_namespace": "openshift-gitops-operator", "target_namespaces": "", "operator_group": "gitops-operator-group"}
+          {"name": "lifecycle-agent", "source": "redhat-operators-custom", "channel": "stable", "install_namespace": "openshift-lifecycle-agent","target_namespace": "", "operator_group": "openshift-lifecycle-agent"},
+        ]
+      RESERVE_BOOTSTRAP: "false"
+      architecture: amd64
+      ipv4_enabled: "true"
+      ipv6_enabled: "true"
+      masters: "1"
+      workers: "0"
+    test:
+    - chain: telcov10n-functional-deploy
```

Comment:
1. Please move the telcov10n-functional-deploy & install-opeartors to a specific workflow.  
2. In the workflow itself, create pre & post stanzas (like we spoke about my PR), the functional-deploy & install operators should exist in the pre stanza, the post should remain empty at the moment.
3. Please add a "test" stanza in the config, with the test itself only.